src/App.jsx

```diff
@@ -1,26 +1,19 @@
-import './App.css';
-
-import {Filter} from "./components/Filters/Filters";
-import {useState,useEffect} from "react";
+import "./App.css";
 
+//import { Filter } from "./components/Filters/Filters";
+//import { useState, useEffect } from "react";
+import { FilterWithRedux } from "./components/Filters/FiltersWithRedux";
```

Comment:
you should not have filter without redux and normal filters in the same branch . 
This is the code that has to be merged in master branch so remove the old filters . we are moving forward with redux. 

If you want to keep the code with normal filters for reference . Then create a new branch from master `git checkout -b feat/filters-without-redux` this will preserve old code for future refernece